tests/lax_numpy_test.py

```diff
@@ -4189,6 +4189,10 @@ def testWhereOneArgument(self, shape, dtype):
     args_maker = lambda: [rng(shape, dtype)]
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=False)
 
+    # JIT compilation requires specifying the size statically:
+    jnp_fun = lambda x: jnp.where(x, size=np.size(x) // 2)
```

Comment:
Is `size(x) // 2` guaranteed sufficient under all random draws of the argument? If I understand correctly, [that's only the expected size](https://github.com/google/jax/blob/648b5d3265c3f1e28be39195a194981239ef2c6d/jax/test_util.py#L728), and there could be fewer zeros.

Reply:
In general `size` may be larger or smaller than the true number, and `jnp.where` should be able to handle that.